src/sentry/eventstream/base.py

```diff
@@ -42,6 +46,16 @@ class GroupState(TypedDict):
 GroupStates = Sequence[GroupState]
 
 
+class EventStreamEventType(Enum):
+    """
+    We have 3 broad categories of event types that we care about in eventstream.
+    """
+
+    Error = "error"  # error, default, various security errors
+    Transaction = "transaction"  # transactions
+    IssuePlatform = "issue_platform"  # events ingested via the issue platform
```

Comment:
i don't really love the naming convention here, Event and Transaction clearly define the type of event and data that we expect, whereas IssuePlatform describes the way we ingested it. that being said introducing a new term here seems like it would just cause more confusion. `OccurrenceEvent` might work

Reply:
Events passed via the issue platform will need a type associated with them. I was planning on having that type match up to the name here. My current thought is that these events coming through the issue platform will have a type of `platform`, or `issue_platform`. I'm not sure if we should make them `occurrences` because then that's conflated with the separate `IssueOccurrence` data. Will think about this a bit more anyway

Reply:
yeah it's more like Error is an event that represents an exception in the code, Transaction is a event that touches the database (or some such thing), and then the last one is an event that represents ...? agree that `IssuePlatform` makes the most sense but that's really a descriptor of how we ingested/created the event not really what the event is capturing.

Reply:
fwiw i went with `platform` but can change it

Reply:
It's tricky because these events won't come from any one source. They could be profiles, replays, etc. In the future if/when we move everything to the platform they could also be errors/transactions as well.

Reply:
I kinda agree the naming is a bit clunky and doesn't fit neatly with errors/transactions, but `issue_platform` makes sense to me since we own this whole code flow piece. I

Reply:
Yeah, so we aren't really locked into this or anything, if we think of something better later it's not a lot of work to refactor

Reply:
We could call it "generic" maybe? I've been doing that in other places.

Reply:
I actually don't hate that... we could have our events be `generic` rather than `platform` as well, which probably makes more sense. What do you think @rjo100, @barkbarkimashark?

Reply:
i like `generic` tbh